pkg/service/handler.go

```diff
@@ -196,6 +201,47 @@ func (h *Handler) GetPProf(ctx context.Context, req *ipc.PProfRequest) (*ipc.PPr
 	}, nil
 }
 
+// GetMetrics implement the handler-side gathering of metrics to return over IPC
+func (h *Handler) GetMetrics(ctx context.Context, req *ipc.MetricsRequest) (*ipc.MetricsResponse, error) {
+	ctx, span := tracer.Start(ctx, "Handler.GetMetrics")
+	defer span.End()
+
+	metrics, err := prometheus.DefaultGatherer.Gather()
+	if err != nil {
+		return nil, err
+	}
+
+	logger.Debugw("returning metrics from handler process", "sizeOfFamilies", len(metrics))
+	metricsAsString, cnt, err := renderMetrics(metrics)
+	if err != nil {
+		return &ipc.MetricsResponse{
+			Metrics: "",
+		}, err
+	}
+	logger.Debugw("Metrics returned from handler process", "cnt", cnt, "metrics", metricsAsString)
```

Comment:
nit: lower case log messages here and below in renderMetrics

Reply:
made message start with lower case (and below as well)